src/cogniteClient.ts

```diff
@@ -289,14 +289,32 @@ export default class CogniteClient {
    * ```
    */
   public getMetadata = (value: any) => this.metadataMap.get(value);
+  public get = (path: string, options?: BaseRequestOptions) =>
+    this.doRawRequest('get', path, options);
+
+  public put = (path: string, options?: BaseRequestOptions) =>
+    this.doRawRequest('put', path, options);
+
+  public post = (path: string, options?: BaseRequestOptions) =>
+    this.doRawRequest('post', path, options);
+
```

Comment:
I would suggest, to make some typedoc `/** */` comments with explanation about each function to show it in our sdk docs later. Check how we document regular endpoints

Reply:
Done, but could be checked in case it should contain some more info